src/goto-programs/goto_program_template.h

```diff
@@ -80,60 +79,129 @@ class goto_program_templatet
     return *this;
   }
 
-  /*! \brief Container for an instruction of the goto-program
-  */
+  /// This class represents a node in the control flow graph of the function.
```

Comment:
They aren't at all similar representations. A `cfgt` is a graph over goto programs, which implies it has references into goto programs.